tests/test_exe.py

```diff
@@ -76,7 +72,7 @@ def test_exe_can_access_xem_script_and_firmware_folders():
     response = requests.get(f"{get_api_endpoint()}shutdown")
     assert response.status_code == 200
 
-    time.sleep(5)  # wait for everything to fully shut down
+    time.sleep(7)  # wait for everything to fully shut down
```

Comment:
definite need growing in the future for a refactor to how shutdown works to have it wait to return the response until shutdown is fully complete.  then it can either return 200 if everything soft stopped correctly, or some error code if it needed to hard stop things. then no need to sleep like this

Reply:
I think we have a TODO for this